### PR TITLE
Appends the CONTRIBUTING.md Banned Content list

### PR DESCRIPTION
**Pull request by Iamgoofball**
no lemons

### CODE REVIEW

**Comment by Cheridan** [comment]
Thank you goof

**Comment by Firecage** [comment]
Mmm. If this will get people to shut up, I like this. Good job Goof.

I do wonder why the other part of the banned content is there anyway. Issues which would arise for people in Germany?

**Comment by bgobandit** [comment]
iirc it was an errorage thing

**Comment by Firecage** [comment]
....Oh....that doesn't surprise me then.

**Comment by Iamgoofball** [comment]
no we didnt want any national socialist party of germany content

**Comment by optimumtact** [comment]
This is just stupid

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -283,6 +283,7 @@ There is no strict process when it comes to merging pull requests, pull requests
 ## Banned content
 Do not add any of the following in a Pull Request or risk getting the PR closed:
  - National Socialist Party of Germany content, National Socialist Party of Germany related content, or National Socialist Party of Germany references
```
Comment (xxalpha) [inline CONTRIBUTING.md]: what about social nationalist?

**Comment by Iamgoofball** [comment]
@optimumtact cheridan declared a lemon ban

**Pull request closed without merging**